flink-contrib/flink-storm/src/main/java/org/apache/flink/storm/wrappers/WrapperSetupHelper.java

```diff
@@ -150,7 +153,7 @@ static synchronized TopologyContext createTopologyContext(
 			}
 			stormTopology = new StormTopology(spouts, bolts, new HashMap<String, StateSpoutSpec>());
 
-			taskId = context.getIndexOfThisSubtask();
+			taskId = context.getIndexOfThisSubtask() + 1;
 
```

Comment:
Actually, it doesn't matter. I set this before changing the topology parsing logic. For some topologies it would only run with this fix. But this has been fixed so the +1 is not necessary anymore.